packages/decdk/bin/decdk-schema.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
These decdk files should not be here.

Reply:
Maybe this is fault of conflicts from a previous merge. Should I delete the file?

Reply:
Ok, I see what you mean. All decdk files should be rolled back to master state. I really don't know why these files ended up here, I did not manually edited them. I will fix this immediately.

Reply:
Yes, please. `decdk` has been moved to a different repository.

Reply:
Done.